scar/main/__version__.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 """version here"""
 
-__version__ = "0.3.0"
+__version__ = "0.3.1"
```

Comment:
I think this should become 0.3.2

Reply:
I hope, the semantic-release will automatically change it when merging the PR.

Check it here, the bot changed it from 0.3.0 -> 0.3.1
https://github.com/Novartis/scAR/pull/31/commits/b3abead0fe746dec4129871fbe734ecb4d867755

If everything goes right, after this PR, it may be pushed to 0.3.3, as there are two commit messages which will trigger the bot. I got no ideas how to avoid these, as it was a testing. But the commit message records are there. So I may just leave it.